app/controllers/album.js

```diff
@@ -0,0 +1,36 @@
+const axios = require('axios');
+const logger = require('../../app/logger');
+const PREVIOUS_MESSAGE = 'Making request..';
+const MESSAGE_OK = 'Request made correctly, sending ...';
+
+const getAlbums = async (req, res) => {
+  const urlAlbums = 'https://jsonplaceholder.typicode.com/albums';
+
+  try {
+    logger.info(PREVIOUS_MESSAGE);
+    const { data } = await axios.get(urlAlbums);
+    logger.info(MESSAGE_OK);
+    res.status(200).send(data);
+  } catch (error) {
+    res.send(error.message);
```

Comment:
In general, we throw our errors and do not throw the same API error. Create specific error for API fail

Reply:
Check how to use the middleware error function